apps/total-typescript/schemas/objects/body.tsx

```diff
@@ -64,6 +64,22 @@ export default defineField({
               },
             ],
           }),
+          // Possible addition! To be discussed
+          // defineArrayMember({
+          //   name: 'conceptLink',
+          //   type: 'object',
+          //   title: 'Concept link',
+          //   icon: HiLink,
+          //   fields: [
+          //     {
+          //       name: 'concept',
+          //       type: 'reference',
+          //       title: 'Concept',
+          //       to: [{type: 'skosConcept'}],
+          //       validation: (Rule) => Rule.required(),
+          //     },
+          //   ],
+          // }),
```

Comment:
Interested in the idea of having links to these concept pages available as a mark for the body text in exercises. This would potentially let us pull in summaries for the concept that were available on hover.

Ofc, the easier version of this would be to just have them as links and maintain the links ourselves if the slugs change.